client/components/deposits-overview/index.tsx

```diff
@@ -2,31 +2,63 @@
  * External dependencies
  */
 import * as React from 'react';
-import { Card, CardHeader } from '@wordpress/components';
+import { Card, CardHeader, ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
 
 /**
  * Internal dependencies.
  */
 import { useAllDepositsOverviews } from 'wcpay/data';
 import strings from './strings';
 import NextDepositDetails from './next-deposit';
+import BannerNotice from 'wcpay/components/banner-notice';
 import DepositsOverviewFooter from './footer';
+import './style.scss';
 
 const DepositsOverview = (): JSX.Element => {
 	const {
 		overviews,
 		isLoading,
 	} = useAllDepositsOverviews() as AccountOverview.OverviewsResponse;
 
+	const completedWaitingPeriod = (
+		wcpaySettings.accountStatus.deposits || {}
+	).completed_waiting_period;
+
 	const { currencies } = overviews;
 
 	const overview = currencies[ 0 ]; // TODO: To handle multiple currencies we'll need to fetch the currently selected currency.
 
+	const userHasNotFinishedNewAccountWaitingPeriodNotice = createInterpolateElement(
+		/* translators: <link> - link to WCPay deposit schedule docs. */
+		__(
+			'Your first deposit is held for seven business days. <link>Why?</link>',
+			'woocommerce-payments'
+		),
+		{
+			link: (
+				<ExternalLink href="https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/#section-1" />
+			),
+		}
+	);
+
 	return (
-		<Card>
+		<Card className="wcpay-deposits-overview">
```

Comment:
I need this so I can overrule the CSS for
`.wcpay-overview .components-notice`.

![Screenshot 2023-03-29 at 03 03 04](https://user-images.githubusercontent.com/73803630/228354113-6c4d40d6-78a4-4396-8513-ff36592a3c2e.png)

Reply:
Since https://github.com/Automattic/woocommerce-payments/pull/5955 [yields](https://github.com/Automattic/woocommerce-payments/pull/5955#issuecomment-1496828581) for other PRs to merge first, I do not make any related change in this PR.